packages/e2e-test-utils/src/templates.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { rest, batch } from './rest-api';
+
+const PATH_MAPPING = {
+	wp_template: '/wp/v2/templates',
+	wp_template_part: '/wp/v2/template-parts',
+};
+
+/**
+ * Delete all the templates of given type.
+ *
+ * @param {('wp_template'|'wp_template_part')} type - Template type to delete.
+ */
+export async function deleteAllTemplates( type ) {
+	const path = PATH_MAPPING[ type ];
+
+	if ( ! path ) {
+		return;
```

Comment:
Instead of just silently return nothing, maybe we should throw an error for unsupported template type.

Reply:
Added in [f2d3f4e](https://github.com/WordPress/gutenberg/pull/38524/commits/f2d3f4e32730d50efa93a97963f7c9b563d8deb1).